app/models/businessTaxReliefs/OtherReliefs.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.businessTaxReliefs
+
+import play.api.libs.json.{Json, OFormat}
+
+case class OtherReliefs(qualifyingLoanInterestPayments: Seq[QualifyingLoanInterestPayments])
```

Comment:
Making this an `Option[Seq[...]]` would probably make life easier later. 

Because we wouldn't want to send an empty Seq we'd want not to send the JSON at all.

Reply:
I think we'll need it to be an Option, because for now we're following that same pattern in the Design where we're calling it three times off each page separately. Because there was no time to redesign the User Journey to make this a single API call.

That, or, we'd need custom JSON writes that don't write `qualifyingLoanInterestPayments: []` and instead just don't output it.